cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,37 @@
+import sys
+import time
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    # help = "Blocks until the database is available"
```

Comment:
Consider uncommenting the `help` attribute to provide a description of the command. This will improve the usability of the command by providing users with information about its purpose.